.github/CODE_OF_CONDUCT.adoc

```diff
@@ -0,0 +1,47 @@
+= Community Code of Conduct
+
+**Version 1.2
+August 19, 2020**
```

Comment:
There is an updated version of the [Community Code of Conduct][coc-html] (and a [markdown version][coc-markdown]).

While it's useful to be consistent in using asciidoc, if we're including standard documents from the eclipse foundation which are only for display within github, I think it's ok to keep them in the original format to make it easier to keep them updated.

[coc-html]: https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php
[coc-markdown]: https://raw.githubusercontent.com/eclipse/.github/master/CODE_OF_CONDUCT.md